projects/addon-mobile/components/mobile-calendar/mobile-calendar.template.html

```diff
@@ -1,4 +1,7 @@
-<header class="t-header">
+<header
+    *ngIf="chooseDayOrRangeTexts$ | async as texts"
```

Comment:
Why only `chooseDayOrRangeTexts` ? Should we check also `closeWord$` & `cancelWord$` ?

Reply:
I think not, because by typing there can not be null. And these are common variables that can be used in other places.